store/mongo/datastore_mongo.go

```diff
@@ -1680,16 +1680,20 @@ func (db *DataStoreMongo) Find(ctx context.Context,
 		options.SetLimit(int64(match.Limit))
 	}
 
+	count, err := collDpl.CountDocuments(ctx, query)
+	if err != nil {
+		count = -1
```

Comment:
@merlin-northern if count fails, return the error, I think it is the right thing to do

Reply:
[my pleasure](https://github.com/mendersoftware/deployments/pull/422/commits/b93353ff381b40541cd3866a83589bafc9a8325f#diff-ea674f2e657b955202c2d74a4c7dec51R1685). fixed.